san/transform.py

```diff
@@ -0,0 +1,31 @@
+import operator
+from san.query import convert_to_datetime_idx_df
+from functools import reduce
+
+QUERY_PATH_MAP = {
+    'eth_top_transactions': ['ethTopTransactions']
+}
+
+def path_to_data(idx, query, data):
+    """
+    With this function we jump straight onto the key from the dataframe, that we want and start from there
+    """
+    return reduce(operator.getitem, ['query_'+str(idx),]+QUERY_PATH_MAP[query], data)
+
+def transform_query_result(idx, query, data):
+    # If there is a transforming function for this query, then the result is
+    # passed for it for another transformation
```

Comment:
Keep the heredoc comment style (""" .... """)

Reply:
You're right, just did that